api/infrastructure/parallelcluster-api.yaml

```diff
@@ -610,6 +621,20 @@ Resources:
                 iam:AWSServiceName:
                   - fsx.amazonaws.com
                   - s3.data-source.lustre.fsx.amazonaws.com
+          - Action:
```

Comment:
What about adding a self explanatory Sid on the newly added statements?

Reply:
Done